sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Azure.Containers.ContainerRegistry.csproj

```diff
@@ -6,6 +6,11 @@
     <PackageTags>Azure Container Registry;$(PackageCommonTags)</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="Models\**" />
+    <EmbeddedResource Remove="Models\**" />
+    <None Remove="Models\**" />
```

Comment:
Just curious why we are doing this.

Reply:
I don't have any files in Models right now, but when I do I will add this back.  I can leave it the way it is if it's cleaner and creates less churn.